transaction/core/src/amount/commitment.rs

```diff
@@ -1,5 +1,5 @@
 use crate::{
-    ring_signature::{Error, Scalar, GENERATORS},
+    ring_signature::{Error, PedersenGens, Scalar},
```

Comment:
We are just importing this name from the Bulletproofs crate that you worked on:

```
pub use bulletproofs_og::{BulletproofGens, PedersenGens};
```

I'm inclined to just use the upstream name to avoid confusion

Reply:
https://github.com/dalek-cryptography/bulletproofs/blob/1a10ce1a5b87299014658770346b376a858e7691/src/generators.rs#L30